internal/request/graphql/schema/types/types.go

```diff
@@ -39,7 +39,7 @@ const (
 	IndexDirectivePropDirection = "direction"
 	IndexDirectivePropIncludes  = "includes"
 
-	IndexFieldInputName      = "name"
+	IndexFieldInputName      = "field"
```

Comment:
question: `IndexFieldInputName` should be `InputFieldInputField` to match the other contants naming convention. You preferred keeping it as `IndexFieldInputName`, can you explain why?

Reply:
It seemed more confusing to me if changed.

Reply:
How about 
```go
IndexDirectiveFieldLabel = "field"
IndexDirectiveFieldDirection = "direction"
```
or maybe just
```go
IncludesPropField = field
IncludesPropDirection = direction
```

Reply:
Updated to your second suggestion